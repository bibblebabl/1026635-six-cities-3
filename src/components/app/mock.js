const offers = [
  {
    id: 1,
    city: {
      name: `Paris`,
      location: {
        x: 52.3909553943508,
        y: 4.85309666406198
      }
    },
    title: `Beautiful & luxurious studio at great location`,
    image: `img/apartment-01.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.`,
    images: [
      `img/room.jpg`,
      `img/apartment-01.jpg`,
      `img/apartment-02.jpg`,
      `img/apartment-03.jpg`,
      `img/studio-01.jpg`,
      `img/studio-photos.jpg`,
    ],
    facilities: [
      `Wi-Fi`,
      `Washing machine`,
      `Towels`,
      `Heating`,
      `Coffee machine`,
      `Baby seat`,
      `Kitchen`,
      `Dishwasher`,
      `Cabel TV`,
      `Fridge`,
    ],
    price: 120,
    rating: 4.8,
    type: `apartment`,
    isFavorite: false,
    isPremium: false,
    bedrooms: 3,
    maxAdults: 4,
    host: {
      name: `Angelina`,
      avatar: `img/avatar-angelina.jpg`,
    }
  },
  {
    id: 2,
    city: {
      name: `Cologne`,
      location: {
        x: 52.3909553943508,
        y: 4.85309666406198
      }
    },
    title: `Beautiful & luxurious studio at great location`,
    image: `img/apartment-02.jpg`,
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.`,
    images: [
      `img/room.jpg`,
      `img/apartment-01.jpg`,
      `img/apartment-02.jpg`,
      `img/apartment-03.jpg`,
      `img/studio-01.jpg`,
      `img/studio-photos.jpg`,
    ],
    facilities: [
      `Wi-Fi`,
      `Washing machine`,
      `Towels`,
      `Heating`,
      `Coffee machine`,
      `Baby seat`,
      `Kitchen`,
      `Dishwasher`,
      `Cabel TV`,
      `Fridge`,
    ],
    price: 120,
    rating: 4.8,
    type: `apartment`,
    isFavorite: true,
    isPremium: false,
    bedrooms: 3,
    maxAdults: 4,
    host: {
      name: ``,
      avatar: `img/avatar-angelina.jpg`,
    }
  },
];

const reviews = [
  {
    "comment": `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    "date": `2019-05-08T14:13:56.569Z`,
    "id": 1,
    "rating": 4,
    "user": {
      "avatarUrl": `img/avatar-max.jpg`,
      "id": 4,
      "isPro": false,
      "name": `Max`
    }
  }
];

export {
  offers,
  reviews,
};
