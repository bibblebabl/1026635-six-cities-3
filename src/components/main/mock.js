export const offers = [
  {
    id: 1,
    city: `Amsterdam`,
    title: `Beautiful & luxurious studio at great location`,
    image: `img/apartment-01.jpg`,
    price: 120,
    rating: 4.8,
    type: `apartment`,
    isFavorite: false,
    isPremium: false,
  },
  {
    id: 2,
    city: `Amsterdam`,
    title: `Beautiful & luxurious studio at great location`,
    image: `img/apartment-02.jpg`,
    price: 120,
    rating: 4.8,
    type: `apartment`,
    isFavorite: true,
    isPremium: false,
  },
];
