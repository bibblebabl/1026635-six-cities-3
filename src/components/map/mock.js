export const offersLocations = [
  {
    "id": 12,
    "city": {
      "name": `Amsterdam`,
      "location": {
        "x": 52.37454,
        "y": 4.897976,
        "zoom": 13
      }
    },
    "location": {
      x: 52.3909553943508,
      y: 4.85309666406198
    }
  },
  {
    "id": 13,
    "city": {
      "name": `Amsterdam`,
      "location": {
        "x": 52.37454,
        "y": 4.897976,
        "zoom": 13
      }
    },
    "location": {
      x: 52.3909553943508,
      y: 4.85309666406198
    }
  },
  {
    "id": 14,
    "city": {
      "name": `Amsterdam`,
      "location": {
        "x": 52.37454,
        "y": 4.897976,
        "zoom": 13
      }
    },
    "location": {
      x: 52.369553943508,
      y: 4.85309666406198
    }
  },
  {
    "id": 15,
    "city": {
      "name": `Amsterdam`,
      "location": {
        "x": 52.37454,
        "y": 4.897976,
        "zoom": 13
      }
    },
    "location": {
      x: 52.3909553943508,
      y: 4.929309666406198
    }
  },
  {
    "id": 16,
    "city": {
      "name": `Amsterdam`,
      "location": {
        "x": 52.37454,
        "y": 4.897976,
        "zoom": 13
      }
    },
    "location": {
      x: 52.3809553943508,
      y: 4.939309666406198
    }
  },
];
