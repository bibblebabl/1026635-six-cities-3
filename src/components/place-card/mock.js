export const offer = {
  id: 2,
  city: {
    name: `Paris`,
    location: {
      x: 52.3909553943508,
      y: 4.85309666406198
    }
  },
  title: `Beautiful & luxurious studio at great location`,
  image: `img/apartment-02.jpg`,
  description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century.`,
  images: [
    `img/room.jpg`,
    `img/apartment-01.jpg`,
    `img/apartment-02.jpg`,
    `img/apartment-03.jpg`,
    `img/studio-01.jpg`,
    `img/studio-photos.jpg`,
  ],
  facilities: [
    `Wi-Fi`,
    `Washing machine`,
    `Towels`,
    `Heating`,
    `Coffee machine`,
    `Baby seat`,
    `Kitchen`,
    `Dishwasher`,
    `Cabel TV`,
    `Fridge`,
  ],
  price: 120,
  rating: 4.8,
  type: `apartment`,
  isFavorite: true,
  isPremium: false,
  bedrooms: 3,
  maxAdults: 4,
  host: {
    name: ``,
    avatar: `img/avatar-angelina.jpg`,
  }
};
