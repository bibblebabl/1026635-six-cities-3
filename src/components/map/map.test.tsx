import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Map from './map';

import {offersLocations} from './mock';

const props = {
  offersLocations,
  selectedCity: {
    "name": `Amsterdam`,
    "location": {
      "x": 52.37454,
      "y": 4.897976,
      "zoom": 13
    }
  },
  hoveredOfferId: 15
};

it(`<Map /> renders correctly`, () => {
  document.body.innerHTML = `<div id="map"></div>`;

  const component = renderer.create(
      <Map {...props} />,
      {
        createNodeMock: () => document.createElement(`div`)
      })
  .toJSON();

  expect(component).toMatchSnapshot();
});


