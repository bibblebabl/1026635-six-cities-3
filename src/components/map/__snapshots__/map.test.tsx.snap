// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> renders correctly 1`] = `
<section
  className="undefined map"
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
